src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+const { STATUS_MESSAGES } = require('../utils/constants');
+const usersService = require('../services/users.service');
+const expensesService = require('../services/expenses.service');
+
+const getAll = (req, res) => {
+  const {
+    userId,
+    from,
+    to,
+  } = req.query;
+
+  const normalizedURL = new URL('http://localhost:3000/' + req.url);
+  const categories = normalizedURL.searchParams.getAll('categories');
```

Comment:
```suggestion
    to,
    categories,
  } = req.query;
```

Reply:
if there are no `categories` in search params we will get `undefined`
if only 1 `categories` we will get just value (....?categories=1)
And if we have many categories we will get array of values (....?categories=1&categories=2&categories=3)

![image](https://github.com/mate-academy/node_accounting-app-v2/assets/86789161/ea010564-92a2-4116-9fcf-a66739d6d01f)